libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DHCP.c

```diff
@@ -973,41 +1039,36 @@ size_t xOptionsLength = sizeof( ucDHCPDiscoverOptions );
 
 		/* After DHCP has failed to answer, prepare everything to start
 		trying-out LinkLayer IP-addresses, using the random method. */
-		xDHCPData.xDHCPTxTime = xTaskGetTickCount();
+		EP_DHCPData.xDHCPTxTime = xTaskGetTickCount();
 
 		xApplicationGetRandomNumber( &( ulNumbers[ 0 ] ) );
 		xApplicationGetRandomNumber( &( ulNumbers[ 1 ] ) );
-		ucLinkLayerIPAddress[ 0 ] = ( uint8_t )1 + ( uint8_t )( ulNumbers[ 0 ] % 0xFDu );		/* get value 1..254 for IP-address 3rd byte of IP address to try. */
-		ucLinkLayerIPAddress[ 1 ] = ( uint8_t )1 + ( uint8_t )( ulNumbers[ 1 ] % 0xFDu );		/* get value 1..254 for IP-address 4th byte of IP address to try. */
+		ucLinkLayerIPAddress[ 0 ] = ( uint8_t )1 + ( uint8_t )( ulNumbers[ 0 ] % 0xFDU );		/* get value 1..254 for IP-address 3rd byte of IP address to try. */
+		ucLinkLayerIPAddress[ 1 ] = ( uint8_t )1 + ( uint8_t )( ulNumbers[ 1 ] % 0xFDU );		/* get value 1..254 for IP-address 4th byte of IP address to try. */
 
-		xNetworkAddressing.ulGatewayAddress = FreeRTOS_htonl( 0xA9FE0203 );
+		EP_IPv4_SETTINGS.ulGatewayAddress = FreeRTOS_htonl( 0xA9FE0203UL );
```

Comment:
Non blocking.
Similar to comment in previous PR, what is `0xA9FE0203UL`? Should we #define it to something more descriptive.